### ISSUE TITLE
[1.12.2] Cha'garoth Minion Spawns

### ISSUE THREAD

**Issue by Entih** [Shinoow/AbyssalCraft#290]
Hello, writing here to say I've been greatly enjoying the mod as part of the "SevTech" megapack, but appear to have run into a rather substantial issue, or more accurately a suggestion.

In progressing through the Dreadlands, I found myself dying against Cha'garoth, only to discover that, on my return to retrieve my corpse(s), the amount of lesser dreadbeasts in Cha'garoth's chamber had, well, skyrocketed.  In my attempts to deal with the issue, the lesser dreadbeasts merely multiplied exponentially, until even flooding the entire chamber with liquid antimatter could not keep pace with them.  In fact, so many were coalescing into existence that the entire multiplayer server had hit 100% CPU usage, and my client was dropping to less than a frame per second, causing the server to lose several seconds worth of ticks at a time, nearly crashing entirely until I managed to navigate out of the dreadlands.

At this point, only a world edit can save our dreadlands, it seems, as getting close to Cha'garoth's chamber is an impossibility.

As such, I would like to recommend perhaps instituting some form of upper limit on these self-replicating spawn, as it seems for fools like me, they may prove too much, and may bring the server down with me.

**Comment by choekstr** [comment]
Same issue.  Hundreds and hundreds of Greater and Lesser DreadSpawn's, replicating out of control.  That's manageable though but alas it is happening in the overworld near my statues and collectors and there are thousands (no exaggeration) to the point of crushing the world and I have to revert to a backup.  leaving the statues up for even 30 minutes while not logged on and my world is toast from so many spawns in the world.

Example of the one time I was able to get logged in and take a screenshot eventhough the world didn't render in fully:
![image](https://user-images.githubusercontent.com/161085/46052378-d062d200-c103-11e8-8577-e2d346c813dc.png)

**Comment by MuteTiefling** [comment]
I'm currently dealing with this issue right now. I left the Abyssal Waste chunk loaded after setting up a Refined Storage network in there... A few days later, I noticed TPS started to tank. I went through all the likely culprits and found nothing. Then I thought perhaps I had left a spawner set up in one of these dimensions....

There are currently 9400 of these things huddled around my chunk loader and I'm struggling to find what entity I would even need to target to kill them all safely.

I wasn't even in the dimension when this happened. Spawns should not occur when no player is around and they should despawn immediately following normal Minecraft mob spawning/despawning rules.
